### PR TITLE
Created SaveMap class to save map as .txt file

### PR DESCRIPTION
**Pull request by Jolijn2020**
also created test for it

currently only creates file in right place with right name, will enhance later.

copied from original branch, to avoid merging conflicts

### CODE REVIEW

File: src/main/java/app/model/map/SaveMap.java [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package app.model.map;
+
+import java.io.File;
+
+public class SaveMap {
+
+    private final String FILE_PATH = "src/main/java/app/model/map/files/";
```
Comment (S010MON) [inline src/main/java/app/model/map/SaveMap.java:7]: Can you save files in the `src/main/java/resources` folder instead

File: src/test/java/jgfx/javagradlefx/SaveMapTest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+        safeMap.saveMap(map);
+
+        String filePathName = "src/main/java/app/model/map/files/createFileTest.txt";
+
+        try
+        {
+            Scanner scanner = new Scanner(filePathName);
+
+            Path path = Paths.get(filePathName);
+            deleteIfExists(path);
```
Comment (S010MON) [inline src/test/java/jgfx/javagradlefx/SaveMapTest.java:35]: When testing files, It's good practice to write a `@BeforeAll` and `@AfterAll` method that does your build and breakdown for all the tests, then you only have to do one of these deletes in the file

**Review by S010MON — CHANGES_REQUESTED** [review]

File: src/main/java/app/controller/Settings.java [hunk 3]
```diff
@@ -8,6 +8,7 @@
 public class Settings
 {
     private boolean unlocked = true;
+    private String name;
```
[the remark this answers is not in the document]
Comment (Jolijn2020) [reply src/main/java/app/controller/Settings.java:11]: added it to FileParser

**Comment by Jolijn2020** [comment]
implemented requested changes

**Review by S010MON — APPROVED** [review]

**Pull request merged**